playback/src/audio_backend/jackaudio.rs

```diff
@@ -47,8 +47,8 @@ impl Open for JackSink {
         let client_name = client_name.unwrap_or_else(|| "librespot".to_string());
         let (client, _status) =
             Client::new(&client_name[..], ClientOptions::NO_START_SERVER).unwrap();
-        let ch_r = client.register_port("out_0", AudioOut::default()).unwrap();
-        let ch_l = client.register_port("out_1", AudioOut::default()).unwrap();
+        let ch_r = client.register_port("out_0", AudioOut).unwrap();
```

Comment:
This is to fix clippy error in https://github.com/librespot-org/librespot/actions/runs/5544745826